kinematics_base_test/src/test_kinematics_as_plugin.cpp

```diff
@@ -1,343 +0,0 @@
- /*********************************************************************
- * Software License Agreement (BSD License)
- *
```

Comment:
This file existed in the PR2 repo at first and it was used as "inspiration" for the **test_kinematics_plugins.cpp** file.  However it was never used in this repo.